app/services/solidus_klarna_payments/create_customer_token_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module SolidusKlarnaPayments
+  class CreateCustomerTokenService < BaseService
+    def initialize(order:, authorization_token:, region:)
+      @order = order
+      @authorization_token = authorization_token
+      @region = region
```

Comment:
We can default the region to 'us' since it is the default when creating the payment method.

Reply:
I think we won't have any benefits customizing it here.

I prefer to avoid customizing it here so if the platform owner wants to change the `default` region it will be enough to change it there.